### PR TITLE
implemented Quiz logic.

### PR DESCRIPTION
**Pull request by DevilStiffer**

### CODE REVIEW

File: Quizzler-iOS13/ViewController.swift [hunk 1]
```diff
@@ -10,11 +10,39 @@ import UIKit
...
         super.viewDidLoad()
         // Do any additional setup after loading the view.
+        
+    
+    }
+    @IBAction func trueButtonAction(_ sender: Any) {
+        updateUI(isNext: true)
+    }
+    
+    @IBAction func falseButtonAction(_ sender: Any) {
```
Comment (vikaspandit89) [inline Quizzler-iOS13/ViewController.swift:27]: Please make sure code should be properly aligned. And removed extra lines

File: Quizzler-iOS13/ViewController.swift [hunk 2]
```diff
@@ -10,11 +10,39 @@ import UIKit
...
+        
+        updateUI(isNext: false)
+        
+    }
+    
+    func updateUI(isNext : Bool) {
+
```
Comment (vikaspandit89) [inline Quizzler-iOS13/ViewController.swift]: Please make sure code should be properly aligned. And removed extra lines

**Review by vikaspandit89 — CHANGES_REQUESTED** [review]

File: Quizzler-iOS13/ViewController.swift (hunk 1 above)
Comment (DevilStiffer) [reply Quizzler-iOS13/ViewController.swift:27]: Done changes.

Comment (DevilStiffer) [reply Quizzler-iOS13/ViewController.swift:27]: Done changes.

File: Quizzler-iOS13/ViewController.swift (hunk 2 above)
Comment (DevilStiffer) [reply Quizzler-iOS13/ViewController.swift]: Done changes.

**Review by vikaspandit89 — APPROVED** [review]
Good to go.

**Pull request merged**